contrail_api_cli_extra/clean/fip.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from sys import exit
+import argparse
+
+from contrail_api_cli.command import Option, Arg
+from contrail_api_cli.utils import printo
+from contrail_api_cli.resource import Collection, Resource
+from contrail_api_cli.exceptions import ResourceNotFound, CollectionNotFound
+
+from ..utils import server_type, CheckCommand
+from refs import CleanRefs
+
+class CleanBrokenFips(CheckCommand):
+    """
+    """
+    description = "Clean for broken floating IPs"
+
+    pool = Option(help="floating IP pool ref",
+                  nargs="?")
+    fip_refs = Arg(help="list of refs [fip0, fip1, fip2, fip3, ...]",
+                   nargs="*", default=[])
+    cassandra_servers = Option(help="cassandra server list' (default: %(default)s)",
+                               nargs='+',
+                               type=server_type,
+                               default=['localhost:9160'])
+
+    def _clean_refs(self, src, tgt, **kwargs):
+        try:
+            fip = Resource('floating-ip', uuid=tgt, fetch=True)
+        except ResourceNotFound:
+            CleanRefs(ref_type="children",
+                      target_type="floating_ip",
+                      paths=[src, tgt],
+                      **kwargs)
+
+    def __call__(self, pool=None, fip_refs=None, cassandra_servers=None, **kwargs):
+        super(CleanBrokenFips, self).__call__(**kwargs)
+        filters=None
+        if pool:
+            filters=[('uuid', pool)]
+        pools = Collection('floating-ip-pool', fetch=True, filters=filters)
```

Comment:
Why not using `contrail_api_cli_extra.utils.PathCommand` class with `resource_type = "floating-ip-pool"` ?